src/vm/gcenv.ee.cpp

```diff
@@ -1176,13 +1176,13 @@ void GCToEEInterface::DiagWalkFReachableObjects(void* gcContext)
 // don't get confused.
 void WalkMovedReferences(uint8_t* begin, uint8_t* end, 
                          ptrdiff_t reloc,
-                         size_t context, 
-                         BOOL fCompacting,
-                         BOOL fBGC)
+                         void* context, 
+                         bool fCompacting,
+                         bool fBGC)
```

Comment:
>bool fBGC [](start = 25, length = 9)

I see you've changed these BOOLs to bools, but how this is called in gc.cpp remains unchanged. this is called in methods that gc_heap::walk_survivors calls, eg, in gc_heap::walk_survivors_for_bgc we are calling this method:

            fn (plug_start, 
                plug_end,
                0,              // Reloc distance == 0 as this is non-compacting
                profiling_context,
                FALSE,          // Non-compacting
                TRUE);          // BGC

arguably you could leave this method's arg types unchanged as it's not strictly on the interface...